jsm.go

```diff
@@ -638,9 +638,12 @@ func (js *js) AddStream(cfg *StreamConfig, opts ...JSOpt) (*StreamInfo, error) {
 }
 
 type (
-	// StreamInfoRequest contains additional option to return details about messages deleted from a stream
+	// StreamInfoRequest contains additional option to return
 	StreamInfoRequest struct {
+		// DeletedDetails when true includes information about deleted messages
 		DeletedDetails bool `json:"deleted_details,omitempty"`
+		// SubjectsFilter when set, returns information on the matched subjects - can be a wildcard or wildcarded subject
```

Comment:
Not sure about the difference between "a wildcard or a wildcarded subject"?

Reply:
">" vs "a.>" - I could just say subject

Reply:
">" is still a subject, so I would not make the distinction.